packages/plugin-core/README.md

```diff
@@ -0,0 +1,17 @@
+# Honeybadger JavaScript Build Plugin
+
+Shared library for uploading sourcemaps and sending deploy notifications to Honeybadger. 
+
+## Usage
+
+You shouldn't need to use this package directly. This package serves as the core implementation of shared functionality meant to be used within Honeybadger's plugins for JS build/bundling systems such as [webpack](https://github.com/honeybadger-io/honeybadger-js/tree/master/packages/webpack) and [rollup](https://github.com/honeybadger-io/honeybadger-js/tree/master/packages/rollup-plugin). 
```

Comment:
@subzero10 I modeled this README on the `core` package, which is available to the public but not encouraged for people to use. I think we actually may find circumstances later in which this may be useful for people to use directly... basically any time we recommend someone hit the sourcemap upload or deploy notification API in JS-land, they could probably use this instead. But I'd say let's use it for our own purposes first, and then decide if we want our users to use it or not later.